eth/fetcher/tx_fetcher.go

```diff
@@ -260,20 +259,9 @@ func (f *TxFetcher) loop() {
 			// Send out all block header requests
 			for peer, hashes := range request {
 				log.Trace("Fetching scheduled transactions", "peer", peer, "txs", hashes)
```

Comment:
the `hashes` here can be quite huge, right? It's not very nice to fill the console with thousands (or even hundreds) of hashes in case the `trace` level is used